solutions.sql

```diff
@@ -0,0 +1,67 @@
+1. get all books by mark lutz
+SELECT books.title FROM books JOIN authors ON books.authorID=authors.id WHERE authors.name ILike '%mark%';
+1. get all books in the childrens books subject
+SELECT books.title FROM books JOIN subjects ON subjects.id=books.subjectId WHERE subjects.name ILIKE '%children%';
+
```

Comment:
This is good work! Try not give every question the number 1 next time but good work Greg looks like your understanding SQL!